src/Workspaces/Core/Portable/Shared/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -55,6 +55,20 @@ public static bool IsSystemVoid([NotNullWhen(returnValue: true)] this ITypeSymbo
         public static bool IsNullable([NotNullWhen(returnValue: true)] this ITypeSymbol? symbol)
             => symbol?.OriginalDefinition.SpecialType == SpecialType.System_Nullable_T;
```

Comment:
```suggestion
            => IsNullable(symbol, out _);
```

Reply:
Okay. I was afraid someone would complain about perf if I did that since I don't know who is referencing it.

Reply:
I'd be fine with going the other way around if you want. It just seems strange to have the `SpecialType` check implemented twice with two different forms.

Reply:
The reason for wanting a different form is that I need to cast to `INamedTypeSymbol`. What do you think about this?

```cs
            if (IsNullable(symbol))
            {
                underlyingType = ((INamedTypeSymbol)symbol).TypeArguments.Single();
                return true;
            }
```

Reply:
such a pity C# doesn't have TS's `IsNullable(this ITypeSymbol? symbol) symbol is INamedTypeSymbol`